src/App.tsx

```diff
@@ -1,5 +1,10 @@
 import styles from 'App.module.scss'
+import WelcomingModal from './components/Modal'
 
 export function App() {
-    return <div className={styles.TODO}>TODO</div>
+    return (
+        <div className={styles.container}>
+            <WelcomingModal />
```

Comment:
we need to show this only when session storage doesnt have a username set. otherwise, i believe its going to keep popping up on page refresh